packages/stylelint-plugin/test.scss

```diff
@@ -5,7 +5,7 @@
 @import "~@kaizen/design-tokens/sass/color-vars";
 @import "././another.scss";
 
-@import "~@kaizen/component-library/styles/layout";
+@import "~@kaizen/deprecated-component-library-helpers/styles/layout";
```

Comment:
There was just one place in this repo that needed to be fixed. This commit was created with `stylelint --fix`